commons/src/main/java/org/infinispan/commons/marshall/WrappedByteArray.java

```diff
@@ -16,16 +16,9 @@
  */
 public class WrappedByteArray implements WrappedBytes {
    private final byte[] bytes;
-   private transient int hashCode;
```

Comment:
Although this still would slow down off heap quite a bit since the hashCode would have to be computed again multiple times.